packages/bsky/src/services/indexing/index.ts

```diff
@@ -127,25 +127,38 @@ export class IndexingService {
     if (!needsReindex) {
       return
     }
-    const { handle } = await this.idResolver.did.resolveAtprotoData(did, true)
-    const handleToDid = await this.idResolver.handle.resolve(handle)
-    if (did !== handleToDid) {
-      return // No bidirectional link between did and handle
+    const atpData = await this.idResolver.did.resolveAtprotoData(did, true)
+    const handleToDid = await this.idResolver.handle.resolve(atpData.handle)
+
+    const handle: string | null = did === handleToDid ? atpData.handle : null
+
+    if (actor && actor.handle !== handle) {
+      const actorWithHandle =
+        handle !== null
+          ? await this.db.db
+              .selectFrom('actor')
+              .where('handle', '=', handle)
```

Comment:
Just double-checking that `handle` should be normalized here (e.g. not going to miss a match due to casing differences).

Reply:
that's a good point - the handle comes from the did doc which could contain capitals. I added an extra normalization step on the incoming handle 👍 

so any handle in the db should be all lower